api/core/management/commands/waitfordb.py

```diff
@@ -36,8 +50,26 @@ def handle(self, *args, **options):
                 with conn.temporary_connection() as cursor:
                     cursor.execute("SELECT 1")
                 logger.info("Successfully connected to the database.")
-                return
+                break
             except OperationalError:
                 logger.warning("Database not yet ready for connections.")
 
             time.sleep(wait_between_checks)
+
+        if should_wait_for_migrations:
+            logger.info("Checking for applied migrations.")
+
+            while True:
+                if time.monotonic() - start > wait_for:
+                    msg = f"Didn't detect applied migrations for {wait_for} seconds."
+                    logger.error(msg)
+                    exit(msg)
+
+                conn = connections["default"]
```

Comment:
This will not work for analytics db migrations, right?

Reply:
Yes, good catch. Do we want to cover the replicas as well?

Reply:
I don't think we need to cover the replicas, no. We can assume that the replication process will handle that for us.

Reply:
Added a `database` option for the command like in Django's `migrate` command.